src/python/grpcio_tests/tests/unit/BUILD.bazel

```diff
@@ -144,6 +144,7 @@ py2and3_test(
     imports = ["../../"],
     main = "_dynamic_stubs_test.py",
     deps = [
+        ":test_common",
```

Comment:
I might miss this line in the first pass. Why do we need to add this dependency now?

Reply:
For `test_common.running_under_bazel`. This test is skipped under gevent because of module loading and subprocess-related problems.